### ISSUE TITLE
getting credentials: exec: executable aws-iam-authenticator not found

### ISSUE THREAD

**Issue by miketamis** [harbur/kubernetic#321]
Im having an issue where I can't connect to AWS EKS cluster.

```
% which aws-iam-authenticator
/usr/local/bin/aws-iam-authenticator
```
<img width="688" alt="Screen Shot 2021-07-20 at 3 20 06 AM" src="https://user-images.githubusercontent.com/1079931/126184693-fe2c5d1d-b7f3-462a-9ee1-f70e6b5cdbf8.png">
<img width="784" alt="Screen Shot 2021-07-20 at 3 19 45 AM" src="https://user-images.githubusercontent.com/1079931/126184707-658dd7f4-b64e-4315-a24e-d9710eb65386.png">

**Comment by dkapanidis** [comment]
Also what version of Kubernetic are you currently using?

**Comment by miketamis** [comment]
Setting the path manually to /usr/local/bin worked

**Comment by dkapanidis** [comment]
Fixed on version v3.3.0 (released on 22 Feb).

Explained on blog post [here](https://www.kubernetic.com/blog/release-3-3-0):

> Default PATH is now effective
> 
> On Linux and Mac machines it is common to see authentication 3rd party binaries such as aws-iam-authenticator be installed under the /usr/local/bin PATH. The PATH on the Preferences page had this value as default, but it was never effective.
>
> ![image](https://user-images.githubusercontent.com/367397/160589475-4c385ac1-3cbf-4684-8b69-cce032c6c4da.png)

This solves the problem described here as it now effectively looks in the default PATH value when unset.
